### ISSUE TITLE
MAUI images not cross-platform on Windows?

### ISSUE THREAD

**Issue by techfan101** [dotnet/docs-maui#1122]
On this page, in the "Draw an Image" section, there is a notation that `PlatformImage` is not supported on Windows.  It seems crazy to have a "**Multi-Platform** Application UI", from Microsoft, that supports all platforms **except** Microsoft Windows.
 
I've asked this before, but I was hoping, with the release of .NET 7, that this might have changed. However, it appears it is still an issue.

Am I really forced to write platform-specific code for something as simple as drawing an image on a canvas? Is there even a way to do this on Windows?

Any idea if/when there will be a mechanism for drawing images on a canvas that is **truly** multi-platform?

It seems this should be slightly embarrassing for Microsoft...that an example in their documentation has to explicitly exclude their own operating system.

For further examples of the havoc this causes, I direct you to the following page: [.NET MAUI / Graphics / Images](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/graphics/images).

The many, many notes/warnings about the incompatibilities inherent in MAUI image handling reinforce the impression that MAUI is "not ready for prime time". I honestly hope this impression, and the delays resolving it, won't negatively impact its future.

Thanks in advance for your time.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0ca86947-204f-9c45-223d-724fb59b4938
* Version Independent ID: 0ca86947-204f-9c45-223d-724fb59b4938
* Content: [Draw graphical objects - .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/graphics/draw?view=net-maui-7.0)
* Content Source: [docs/user-interface/graphics/draw.md](https://github.com/dotnet/docs-maui/blob/main/docs/user-interface/graphics/draw.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @techfan101 

I hear you and I agree. There have been multiple attempts to fix this issue (see https://github.com/dotnet/Microsoft.Maui.Graphics/pull/464 and https://github.com/dotnet/maui/pull/9948) but it didn't make it in for .NET 7. 

I've sent a message to the engineers to find out how/when this will get resolved. I'll get back to you when I know more.

**Comment by techfan101** [comment]
Thank you for trying @davidbritch. I really do appreciate it. I hope the response is positive.

However, I already waited 6 months for a fix in .NET 7. From the discussion, it looks like we're waiting another year for a fix in .NET 8 (best case). And, I'm not even counting how long I waited for MAUI to hit GA in the first place.

Drawing an image is about as basic as it gets for a UI. I can't believe it takes a year and a half to rename a couple of classes and interfaces. That's crazy!

Basic issues like this coupled with the ever-growing backlog are creating a bad impression in the development community. At this pace, MAUI will fail. It will certainly be great when it's done, but there will be no one left waiting to use it.

I hope Microsoft adds more resources to the project (if that's the problem).

For my part, I started transitioning future development to Flutter. I guess I'll continue along that path, which is heartbreaking.

I genuinely wanted MAUI to succeed. I prefer C# to Dart and VS Pro to VS Code, but I guess it's time to move on.

Maybe I'll check back, in another 6 months, to see if the situation has improved or continues to deteriorate. I might still have one final opportunity to try a project in MAUI :)

Best of luck and thanks again for trying.
